png_to_asm.py

```diff
@@ -3,16 +3,18 @@
 
 Color = Tuple[int, int, int]
 
+mode = ""
 
-color_list = [(0, 0, 0), (255, 0, 0), (0, 255, 0), (0, 0, 255)]
+color_list = []
 
 def load_image(src: str) -> List[Color]:
     """
     Return a list of tiles
     The returned pixels do not have their alpha values!
     """
     image = Image.open(src)
-
+    global mode
+    mode = image.mode
```

Comment:
You don't need this since image _is_ an argument of extract_tiles

Reply:
right, I had'nt noticed that.